_docs-sources/refarch/access/setup-auth/index.md

```diff
@@ -1,27 +1,98 @@
+However, to access the Terraform state containing the password, you need to already be authenticated to the account. Thus to get access to the initial admin IAM User, we will use the root user credentials. To do this, you can either:
 
-## 3. Configure other IAM users 
+* Login on the AWS Web Console using the root user credentials for the security account and use the web console to setup the web console password and AWS Access Keys for the IAM User.
 
-Haxx0r ipsum foo Trojan horse new all your base are belong to us ip error private shell fopen semaphore epoch char packet sniffer segfault gurfle bypass. Memory leak bubble sort injection leet malloc brute force double xss mega sudo mountain dew void echo win emacs linux piggyback bin. I'm compiling float bang case cat infinite loop Donald Knuth unix for /dev/null machine code then chown d00dz worm gnu crack packet bar eof while.
+* Use the [Gruntwork CLI](https://github.com/gruntwork-io/gruntwork/) to rotate the password using the command:
 
-Lib void brute force bypass nak concurrently all your base are belong to us break leapfrog bit default packet sniffer Linus Torvalds. Man pages packet stack trace Starcraft Donald Knuth pwned worm hello world public giga frack gurfle. Irc fork malloc fopen script kiddies flood blob fail hexadecimal while access semaphore loop mega Trojan horse foo gobble.
+```
+gruntwork aws reset-password --iam-user-name <IAM_username>
+```
+
+Once you have access with via you IAM user, finish hardening your security posture:
+
+1. Enable MFA. Follow [these instructions](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable.html) to enable MFA for your IAM User. It takes less than a minute and dramatically improves your security posture. Moreover, MFA is required by the Reference Architecture, and you won't be able to access any other accounts without it! Note that the name of the MFA must be named exactly the same as the AWS IAM Username.
+
+1. Logout and log back in. After enabling MFA, you need to log out and then log back in, thereby forcing AWS to prompt you for an MFA token. Until you do this, you will not be able to access anything else in the web console!
+
+1. Create access keys. Follow [these instructions](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) to create access keys for yourself. Store the access keys in a secrets manager. You will need these to authenticate to AWS from the command-line.
+
+## Step 3. Configure other IAM users 
+
+Now that your IAM user is all set up, you can configure IAM users for the rest of your team. All of the IAM users are managed as code in the security account, in the `account-baseline-app module`. If you open the `terragrunt.hcl` file in that repo, you should see the list of users, which will look something like:
```

Comment:
Should we include the direct path to the file here, or do we think that's too brittle?

```suggestion
Now that your IAM user is all set up, you can configure IAM users for the rest of your team. All of the IAM users are managed as code in the security account in the `account-baseline-app module`. If you open the `terragrunt.hcl` file in that repo, you should see the list of users, which will look something like:
```